src/components/Disputes/DisputeInfoContent.js

```diff
@@ -109,10 +109,7 @@ function Field({ label, loading, value, ...props }) {
         if (typeof value === 'string') {
           const ipfsPath = getIpfsCidFromUri(value)
           if (ipfsPath) {
-            const ipfsUrl = resolvePathname(
-              ipfsPath,
-              `${IPFS_ENDPOINT}/${ipfsPath}`
-            )
+            const ipfsUrl = resolvePathname(ipfsPath, `${IPFS_ENDPOINT}/`)
```

Comment:
Note that this is functionality the same as above (and the above is a bit confusing—why are we appending the same path onto itself?).

Reply:
ahh, yes good catch, seems that the library somehow internally was resolving it right anyways? since the second argument is the current pathname and the first one is the one that you would like to resolve

Reply:
Yes, since resolving `foo` into `bar/baz` results in `bar/foo`.